src/session/session_dhandle.c

```diff
@@ -559,17 +578,20 @@ __wt_session_get_btree_ckpt(WT_SESSION_IMPL *session, const char *uri, const cha
             /*
              * If we have not raced with a checkpoint, we still may have an inconsistency in a
              * specific scenario that involves bulk operations. When a bulk operation finishes, it
-             * generates a single file checkpoint which is different from a system wide checkpoint.
+             * generates a single file checkpoint which is different from a system-wide checkpoint.
              * The single file checkpoint only bumps the data store time which makes it ahead of the
-             * last system wide checkpoint time and leads to the inconsistency.
+             * last system-wide checkpoint time and leads to the inconsistency.
              */
             if (first_snapshot_time == snapshot_time && ds_time > snapshot_time &&
               hs_time <= snapshot_time) {
+
```

Comment:
Remove added newline.

Reply:
Removed in 0568757e36ccd727773af296038921f8a4ba00b0